src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,20 @@
-export const TodoInfo = () => {};
+import cn from 'classnames';
+import { UserInfo } from '../UserInfo';
+import React from 'react';
+import { Task } from '../../types/Task';
+
+type Props = {
+  todo: Task;
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => (
+  <article
+    data-id={todo.id}
+    className={cn('TodoInfo', {
+      'TodoInfo--completed': todo.completed,
+    })}
+  >
+    <h2 className="TodoInfo__title">{todo.title}</h2>
+    <UserInfo user={todo.user} />
+  </article>
```

Comment:
Use destruction for `todo`
```suggestion
  >
    <h2 className="TodoInfo__title">{todo.title}</h2>
    
    <UserInfo user={todo.user} />
  </article>
```